test/functional/validation-options.spec.ts

```diff
@@ -1086,4 +1086,30 @@ describe('context', () => {
       expect(errors[1].contexts['contains']).toEqual({ bye: 'now' });
     });
   });
+
+  it('should stop at first error.', () => {
+    class MyClass {
+      @IsDefined({
+        message: 'isDefined',
+        context: {
+          bye: 'now',
+        },
+      })
+      @Contains('hello', {
+        message: 'String is not valid. You string must contain a hello word',
+        context: {
```

Comment:
Checking context functionality is not part of this test, please remove and add to where context is tested.

Reply:
fixed.